python_modules/dagster-graphql/dagster_graphql/schema/logs/events.py

```diff
@@ -332,6 +334,9 @@ def resolve_metadataEntries(self, _graphene_info):
 
         return _to_metadata_entries(self._metadata.metadata_entries)
 
+    def resolve_expectationResults(self, _graphene_info) -> Sequence[GrapheneExpectationResult]:
+        return "what goes here???"
```

Comment:
@clairelin135 are you able to help a graphql noob (me) with this? How do I turn `ExpectationResult` objects into `GrapheneExpectationResults`?